docs/getting-started.md

```diff
@@ -63,6 +63,8 @@ curl localhost:10101/index/repository -X POST
 ``` response
 {"success":true}
 ```
+Note: 
```

Comment:
I know we do the `Note:` thing above, but the newline after it doesn't actually get rendered. I say we drop the `Note:` line OR if we actually do want a fancy note block, there is an example of that above where we do `<div class="note">`